news/_posts/2020-07-16-journey-to-improve-the-time-to-interactive-metric.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "A journey to improve the Time To Interactive metric in Classic Theme"
+subtitle: "When the lazy loading allow us to improve the LightHouse Score"
+date: 2020-07-16 9:00:00
+authors: [Progi1984]
+image: /assets/images/theme/meta-logo-build.png
+icon: icon-lab
+tags:
+- development
+- ux
+- theme
+---
+
+# A journey to improve the Time To Interactive metric in PrestaShop thanks to the lazy loading
```

Comment:
The first sentence is used as a summary for the article on homepage. Also the `h1` comes from the post `title` so you cant remove it 😉  (AFAIK)